src/WebJobs.Script/Binding/FunctionBinding.cs

```diff
@@ -134,5 +121,13 @@ protected string Resolve(string name)
 
             return _config.HostConfig.NameResolver.ResolveWholeString(name);
         }
+
+        internal static void AddStorageAccountAttribute(Collection<CustomAttributeBuilder> attributes, string connection)
```

Comment:
I asked myself that, and no, I don't think so. I think we'll limit it to just parameter bindings in script, since there is UI for it. We could also support function level if we wanted, but I don't think that is important. And the class level attribute really doesn't translate to script, since we own the container class for all the functions.